generic/lshw.py

```diff
@@ -28,9 +28,12 @@ class Lshwrun(Test):
     mainboard configuration, CPU version and speed,cache configuration, bus
     speed, etc. on DMI-capable x86 or IA-64 systems and on some PowerPC
     machines (PowerMac G4 is known to work).
+
+    :avocado: tags=privileged
     """
     interface = process.system_output("ip route show")
-    active_interface = process.system_output("ifconfig | head -1 | cut -d':' -f1", shell=True).strip().split()[0]
+    active_interface = process.system_output(
+        "ifconfig | head -1 | cut -d':' -f1", shell=True).strip().split()[0]
```

Comment:
@PraveenPenguin `ifconfig` will be deprecated, and `ip` command is preferred. Can you take care of that too here ?

Reply:
sure i will